ui/chat/css/build/chat.mod.scss

```diff
@@ -1,2 +1,3 @@
 @import '../../../common/css/plugin';
 @import '../chat.mod';
+@import '../../../common/css/form/cmn-toggle';
```

Comment:
Hello, this import is probably not at the right place, most likely should be in _chat, you should test viewing as a spectator